xapian-letor/tests/api_letor.cc

```diff
@@ -180,11 +440,147 @@ DEFINE_TESTCASE(listnet_ranker, generated)
 		   ranker.score(qrel, "", "ListNet_Ranker",
 				"scorer_output.txt", 10));
     ranker.score(query, qrel, "ListNet_Ranker", "ndcg_output.txt", 10);
-    ranker.score(query, qrel, "ListNet_Ranker", "err_output.txt", 10, "ERRScore");
+    ranker.score(query, qrel, "ListNet_Ranker", "err_output.txt", 10,
+								"ERRScore");
```

Comment:
Was breaking the line limit of 80, what would be the better way to represent? 
 
```
ranker.score(query, qrel, "ListNet_Ranker", 
            "err_output.txt", 10, "ERRScore"); 
```
@ojwb

Reply:
@addy007-icy the point Ayush was making was that going through and fixing formatting issues as part of other commits/PRs isn't the best way to do things.

To answer your question, that's not quite how I'd do it — align the opening quote on the second line with the 'q' of 'query' in the first. (And I probably would break before `"ERRScore"` and not before `"err_output.txt"` because I think it'll be clearer what's going on compared to the line above.)